frontend/src/layouts/AuthLayout_gdd.vue

```diff
@@ -1,19 +1,181 @@
+                  <div id="popover-target-1" class="pointer">
+                    <b-img src="img/svg/type.svg" width="19" class="svg"></b-img>
+                  </div>
+                  <b-popover
+                    target="popover-target-1"
+                    triggers="click"
+                    placement="top"
+                    variant="dark"
+                  >
+                    <div class="text-light">
+                      <span class="pointer" @click="setTextSize(0.85)">{{ $t('85') }}</span>
+                      {{ $t('|') }}
+                      <span class="pointer" @click="setTextSize(1)">{{ $t('100') }}</span>
+                      {{ $t('|') }}
+                      <span class="pointer" @click="setTextSize(1.25)">{{ $t('125') }}</span>
+                    </div>
+                  </b-popover>
+                </b-col>
+              </b-row>
+              <b-row class="d-inline d-sm-inline d-md-none d-lg-none mb-3">
+                <b-col class="text-center">
+                  <b-avatar src="img/brand/gradido_coin●.png" size="6rem"></b-avatar>
+                  <b-row class="mt-4">
+                    <b-col class="zindex1000 mb--4">
+                      <!-- <navbar-small /> -->
+                    </b-col>
+                  </b-row>
+                </b-col>
+              </b-row>
+              <b-card-body class="">
+                <router-view></router-view>
+              </b-card-body>
+            </b-card>
+          </div>
+          <auth-footer v-if="!$route.meta.hideFooter" class="mt-4 pr-5"></auth-footer>
+        </b-col>
+      </b-row>
+      <!-- <auth-layout-gdd />-->
     </div>
   </div>
 </template>
 <script>
-import ContentFooter from '@/components/ContentFooter.vue'
-import LanguageSwitch from '@/components/LanguageSwitch.vue'
+import MobileStart from '@/components/Auth/MobileStart.vue'
+import Navbar from '@/components/Auth/Navbar.vue'
+import NavbarSmall from '@/components/Auth/NavbarSmall.vue'
+import Carousel from '@/components/Auth/Carousel.vue'
+import LanguageSwitch from '@/components/LanguageSwitch2'
+import AuthFooter from '@/components/Auth/Footer.vue'
+
 export default {
+  name: 'AuthTemplate',
   components: {
-    ContentFooter,
+    MobileStart,
+    Navbar,
+    NavbarSmall,
+    Carousel,
     LanguageSwitch,
+    AuthFooter,
+  },
+  data() {
+    return {
+      mobileStart: window.innerWidth < 1025,
+      windowWidth: window.innerWidth,
+    }
+  },
+  methods: {
+    isMobileStart(boolean) {
+      this.mobileStart = boolean
+    },
+    setTextSize(size) {
+      this.$refs.pageFontSize.style.fontSize = size + 'rem'
+    },
+    onResize() {
+      this.mobileStart = window.innerWidth < 1025
+    },
+  },
+  mounted() {
+    this.$nextTick(() => {
+      window.addEventListener('resize', this.onResize)
+    })
+  },
+  watch: {
+    windowWidth() {
+      window.addEventListener('resize', this.onResize)
```

Comment:
You really want to add the same event listener every time windowWidth changes? I am not sure about this.
I suggest to have `windowWitdh`as computed property. Then we must verify, that `windowWidth` is updated when `window.innerWidth` updates. If this is the case, then we do not need to watch on `windowWidth`

Reply:
this was also my idea at the beginning. but unfortunately i could not watch window.innerWidth

Reply:
is deleted